### PR TITLE
Added firebase database & connected to Discover screen

### PR DESCRIPTION
**Pull request by AlexeiGitH**
Added firebase database to the project. 
Discover screen now fetches workout from the database and shows a bit more info. 
There are still things to do over the next few weeks + polish the design.
Simulator Screen Shot - iPhone X - 2019-06-28 at 17 03 17

### CODE REVIEW

File: screens/Discover.js [hunk 1]
```diff
@@ -1,115 +1,83 @@
...
+      darkTheme: window.darkTheme
+    }
+    this.workouts = []
+    firebase.database().ref('/common/workouts/').once('value').then((snapshot) => {
+      var wrks = []
+      snapshot.forEach(function (workout) {
+        wrks.push(workout.val())
+      });
+      this.workouts = wrks
+      this.setState({ dataReceived: true })
```
Comment (klymenkoo) [inline screens/Discover.js]: You shouldn't call `setState` inside constructor. Move it to `componentDidMount()`

Comment (AlexeiGitH) [reply screens/Discover.js]: As well as fetching data!

Comment (klymenkoo) [reply screens/Discover.js]: Did you push changes? @AlexeiGitH

Comment (AlexeiGitH) [reply screens/Discover.js]: Forgot. Just did.

**Comment by klymenkoo** [comment]
@AlexeiGitH looks good. Did you have a chance to take a look at a pop-up issue? If not, we can merge this one, and fix that one later in another PR.

**Comment by AlexeiGitH** [comment]
I haven't changed the part that is responsible for the pop-up, lets take a look at it later.

**Pull request merged**